### ISSUE TITLE
"Contributions" section in readme outdated

### ISSUE THREAD

**Issue by hansemannn** [tidev/titanium_mobile#13353]
### I have searched and made sure there are no existing issues for the issue I am filing

- [X] I have searched the existing issues

### Description

The "Contributions" section in the README.md is outdated: https://github.com/tidev/titanium_mobile#contributing

### Expected Behavior

The new way should be described.

### Actual behavior

The developer is redirected to the outdated `cla.appcelerator.com` org.

### Reproducible sample

n/a

### Steps to reproduce

n/a

### Platform

CLI

### SDK version you are using

n/a

**Comment by cb1kenobi** [comment]
Yes, we know. Here's the new boilerplate copy we will add to all projects' readmes: https://github.com/tidev/organization-docs/blob/main/BOILERPLATE_README.md. Despite having the copy written, we cannot add it to the readmes yet as the tidev.io pages don't exist. It's gonna take a couple weeks as this and the tidev.io site are lower priority than the CI stuff.
